Library/Formula/ooniprobe.rb

```diff
@@ -0,0 +1,154 @@
+    sha256 "a99db8e59c120138ad8a72eecedcc24b4510d2eed3ce48213b7e32f22cc4ee6e"
+  end
+
+  resource "pypcap" do
+    url "https://pypi.python.org/packages/source/p/pypcap/pypcap-1.1.1.tar.gz"
+    sha256 "b310d5af36f5d68ef4217fda68086ffce56345b415eaac15ad618f94057b017b"
+  end
+
+  resource "PyYAML" do
+    url "https://pypi.python.org/packages/source/P/PyYAML/PyYAML-3.11.tar.gz"
+    sha256 "c36c938a872e5ff494938b33b14aaa156cb439ec67548fcab3535bb78b0846e8"
+  end
+
+  resource "scapy-real" do
+    url "https://pypi.python.org/packages/source/s/scapy-real/scapy-real-2.2.0-dev.tar.gz"
+    sha256 "106a2774e135cd058a78b52d55efdf6ef648a6161ecce3919cc7dbda50babdd4"
+  end
+
+  resource "service_identity" do
+    url "https://pypi.python.org/packages/source/s/service_identity/service_identity-14.0.0.tar.gz"
+    sha256 "3105a319a7c558490666694f599be0c377ad54824eefb404cde4ce49e74a4f5a"
+  end
+
+  resource "six" do
+    url "https://pypi.python.org/packages/source/s/six/six-1.9.0.tar.gz"
+    sha256 "e24052411fc4fbd1f672635537c3fc2330d9481b18c0317695b46259512c91d5"
+  end
+
+  resource "Twisted" do
+    url "https://pypi.python.org/packages/source/T/Twisted/Twisted-15.0.0.tar.bz2"
+    sha256 "ac609262253057cf2aeb9dc049ba7877d646f31b4caef06a50189a023df46b51"
+  end
+
+  resource "txsocksx" do
+    url "https://pypi.python.org/packages/source/t/txsocksx/txsocksx-1.13.0.3.tar.gz"
+    sha256 "df1a9e7062c7e3693c39953705b75e0feb7b8746a05135ffb2b8cd98708c9c43"
+  end
+
+  resource "txtorcon" do
+    url "https://pypi.python.org/packages/source/t/txtorcon/txtorcon-0.12.0.tar.gz"
+    sha256 "206b1bd8a840119c12d9b85d638ab9defec5b376436fa36be9139ab1ebc8cd78"
+  end
+
+  resource "zope.interface" do
+    url "https://pypi.python.org/packages/source/z/zope.interface/zope.interface-4.1.2.tar.gz"
+    sha256 "441fefcac1fbac57c55239452557d3598571ab82395198b2565a29d45d1232f6"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    resources.each do |r|
+      r.stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    # provided by libdnet
+    inreplace "requirements.txt", "pydumbnet", ""
+
+    # force a distutils install
+    inreplace "setup.py", "def run(", "def norun("
+    (buildpath/"ooni/settings.ini").atomic_write <<-EOS.undent
+      [directories]
+      usr_share = #{share}/ooni
+      var_lib = #{var}/lib/ooni
+    EOS
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    man1.install Dir["data/*.1"]
+    (share/"ooni").install Dir["data/*"]
+    (var/"lib/ooni").mkpath
+  end
+
+  def caveats; <<-EOS.undent
+    Run `ooniresources --update-inputs --update-geoip` to fetch fresh data
```

Comment:
The formula can, sure. It hits the network to fetch some datasets. Is that okay? Figure it makes the most sense in post_install.